### PR TITLE
Added All Methods of Navigator state utils

### PR DESCRIPTION
**Pull request by renganatha10**
I have added all methods of state utils as an action and created corresponding reducer,

### CODE REVIEW

**Comment by mpinter** [comment]
This would be nice to have :), any progress on merging and publishing ?

**Comment by lfrodrigues** [comment]
Please merge this, I had to make a fork to be able to use in my app.

**Comment by callmephilip** [comment]
Sorry for the delay guys. @rengasahusoft is there a chance you could add respective unit tests to cover changes that you've made? We will need to get proper test coverage on this before we could merge this

File: .eslintrc [hunk 1]
```diff
@@ -37,7 +37,7 @@
     # Best Practices
     accessor-pairs: 2,
     block-scoped-var: 0,
-    complexity: [2, 6],
```
Comment (callmephilip) [inline .eslintrc:40]: what is the reason for disabling this?

**Comment by callmephilip** [comment]
Closing this in favor of https://github.com/thebakeryio/react-native-navigation-redux-helpers/pull/3 which includes bug fixes and test coverage + docs

**Pull request closed without merging**